AdminWebsite/AdminWebsite.UnitTests/Controllers/StaffMemberControllerTest.cs

```diff
@@ -27,7 +27,7 @@ public void Setup()
         {
             _bookingsApiClient = new Mock<IBookingsApiClient>();
 
-            _controller = new StaffMemberController(_bookingsApiClient.Object, JavaScriptEncoder.Default);
```

Comment:
Have you considered using AutoMock to manage dependencies for unit tests?
It reduces the amount of code you'll need to write when you add or remove a dependency to a class